test/simulator/abilities/pickup.js

```diff
@@ -53,9 +53,9 @@ describe('Pickup', function () {
 			{species: 'Clefable', ability: 'unaware', item: 'ejectbutton', moves: ['followme']},
 			{species: 'Magikarp', ability: 'rattled', moves: ['splash']},
 		]);
-		battle.makeChoices('move shadowsneak, move rest', 'move uturn, move followme');
-		battle.makeChoices('move shadowsneak, move rest', 'switch 3'); // Eject Button activated
-		battle.makeChoices('move shadowsneak, move rest', 'switch 3');
+		battle.makeChoices('move shadowsneak 1, move rest', 'move uturn 1, move followme');
+		battle.makeChoices('', 'switch 3'); // Eject Button activated
+		battle.makeChoices('', 'switch 3');
```

Comment:
As per latest @Zarel's proposal, this line should change to explicity proc the decisions (add the missing command).